ONSdigital/eq-author
# Upgrade React + Other Dependencies

### What is the context of this PR?

Upgrades all dependencies, including React v16.

### How to review 

- ensure app still runs and builds as expected
- ensure storybook builds
